cupy/fft/fft.py

```diff
@@ -703,6 +703,7 @@ def irfft(a, n=None, axis=-1, norm=None):
 
     Args:
         a (cupy.ndarray): Array to be transform.
+            WARNING: May be modified in CUDA 10.1 and above.
```

Comment:
One more thing: we also need to add the warning to `irfftn`.

Reply:
> I think having a docstring block like this would make it more noticeable?
> 
> ```python
> .. warning:: The input array may be modified in CUDA 10.1 and above.
> ```
> 
> You can put it right above `.. seealso::`.

Good idea, will do that shortly.

> One more thing: we also need to add the warning to `irfftn`.

Absolutely, missed that. Btw, do you know if https://github.com/cupy/cupy/blob/4543621001e5925c16ca18648913d6c22d0530bc/cupy/fft/fft.py#L826-L830 refers to this issue? The warning clearly states output, but I'm not sure whether this is a different problem or if the warning message is misleading.

Reply:
That was added for a separate accuracy-related bug and unrelated to the issue of overwriting inputs (see https://github.com/cupy/cupy/issues/2569#issuecomment-571863448 and #2922)

Reply:
Thanks for confirming @grlee77 , I will keep it as is then, I don't have access to Pascal GPUs with CUDA 11.0 now to confirm whether this has been fixed or not.

Reply:
Fixed the warning formatting in https://github.com/cupy/cupy/pull/3763/commits/b4f143fc442ac5a71beb4bcde10448cdefd63703 .